src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -17,6 +17,9 @@
  * The data transfer class for Instructor entities.
  */
 public class InstructorAttributes extends EntityAttributes {
+    // Note: we should disable HTML escaping when creating gson string of the InstructorAttributes
+    // objects because if the name of the instructor contains any html character, we cannot
+    // search for the object by the name anymore.
```

Comment:
On the latest commit, I am also having trouble searching for instructors containing html characters in the name